person.rb

```diff
@@ -1,13 +1,17 @@
 require './nameable'
+require './rental'
 
 class Person < Nameable
   attr_accessor :id, :name, :age
 
+  attr_reader :rental
```

Comment:
- [ ] Getters and Setters for @rentals should be present as `attr_accessor rental`. Kindly fix it